tests/rest/test_rest_config.py

```diff
@@ -11,12 +11,13 @@
 
 from taipy.common.config._config import _Config
 from taipy.common.config.checker.issue_collector import IssueCollector
+from taipy.common.config.config import Config
 from taipy.rest.config.rest_checker import _RestConfigChecker
 from taipy.rest.config.rest_config import RestConfig
 
 
 def test_rest_config_default_values():
-    rest_config = RestConfig()
+    rest_config = Config.configure_rest()
```

Comment:
This line must be unnecessary. The default values should be automatically populated if the the method is not called.

```suggestion
```